config/default.yaml

```diff
@@ -28,6 +28,13 @@ pantalaimon:
   # After successfully logging in once, this will be ignored, so this value can be blanked after first startup.
   password: your_password
 
+# Experimental usage of the matrix-bot-sdk rust crypto.
+# This can not be used with Pantalaimon.
+# Make sure to setup the bot as if you are not using pantalaimon for this.
+#
+# Warning: At this time this is not considered production safe.
```

Comment:
Really? Are we saying that or are the authors of `RustSdkCryptoStorageProvider` saying that?

Reply:
Copied from chat:

> I am saying that one. Its not well tested at all currently so I think it makes sense to first have this as a non prod flag. Also I remember occasional breaking stuff from hookshot users. So I think it makes sense to first have a release or 2 where this is available but not yet guaranteed to be stable since we didnt test it enough imho. And then we can revisit this and drop that sentence I guess.
> 
> Like I am seeing draupnir as something that should run reliably most of the time if not all of the time. And I dont think we have enough data if this is the case with bot-sdk (Though if
> [TravisR](https://matrix.to/#/@travis:t2l.io) thinks this is stable enough then I am also happy to remove the sentence. Its just that I cant with confidence attest this yet since I didnt test it extensively enough :) )

Reply:
https://matrix.to/#/!IaWNErZAgQUhGqJXjX:matrix.org/$hV8HlsBuarqdS7guacNp4Sek8OGGrpf4uqCMSJwx_Qs?via=matrix.org&via=envs.net&via=the-apothecary.club

Reply:
Oh and link for future readers: https://matrix.to/#/!IaWNErZAgQUhGqJXjX:matrix.org/$hV8HlsBuarqdS7guacNp4Sek8OGGrpf4uqCMSJwx_Qs?via=matrix.org&via=envs.net&via=the-apothecary.club